src/modules/processes.nix

```
@@ -35,7 +35,7 @@ let
   implementation-options = config.process.${implementation};
   envList =
     lib.mapAttrsToList
-      (name: value: "${name}=${toString value}")
+      (name: value: ''${name}="${toString value}"'')
```

Might be even safer to use `toJSON`? Not sure what other side-effects `toJSON` will bring that aren't compatible with bash, but it at least makes so that `"` inside the value are escaped.

```suggestion
      (name: value: "${name}=${toJSON value}")
```